translations/fr-fr.yaml

```diff
@@ -371,15 +371,15 @@ media:
         other: "Autre"
   hide-posts:
     text: "Masquer les posts concernant {media}"
-    info: "Les posts n'apparaîtrons plus ni dans votre fil d'abonnements{type, select,
-          Anime {, ni fil d'animé}
-          Manga {, ni fil de manga}
+    info: "Les posts n'apparaîtrons plus ni dans votre fil d'abonnements, ni {type, select,
```

Comment:
This produces the following:

- Les posts n'apparaîtrons plus ni dans votre fil d'abonnements, ni d'animés
- Les posts n'apparaîtrons plus ni dans votre fil d'abonnements, ni de mangas
- Les posts n'apparaîtrons plus ni dans votre fil d'abonnements, ni 

Previous produced:

- Les posts n'apparaîtrons plus ni dans votre fil d'abonnements, ni fil d'animé
- Les posts n'apparaîtrons plus ni dans votre fil d'abonnements, ni fil de manga
- Les posts n'apparaîtrons plus ni dans votre fil d'abonnements

Reply:
Yes, forgot about the other, correcting